docs/source/configuration/advanced_configuration.md

```diff
@@ -262,58 +262,73 @@ Since both of the file names (`catalog.yml` and `catalog_globals.yml`) match the
 #### Other configuration files
 It's also possible to use variable interpolation in configuration files other than parameters and catalog, such as custom spark or mlflow configuration. This works in the same way as variable interpolation in parameter files. You can still use the underscore for the templated values if you want, but it's not mandatory like it is for catalog files.
 
-### How to use custom resolvers in the `OmegaConfigLoader`
-`Omegaconf` provides functionality to [register custom resolvers](https://omegaconf.readthedocs.io/en/2.3_branch/usage.html#resolvers) for templated values. You can use these custom resolves within Kedro by extending the [`OmegaConfigLoader`](/kedro.config.OmegaConfigLoader) class.
+### How to use resolvers in the `OmegaConfigLoader`
+Instead of hard-coding values in your configuration files, you can also dynamically compute them using `OmegaConf`'s
+[resolvers](https://omegaconf.readthedocs.io/en/2.3_branch/custom_resolvers.html#resolvers) functionality. You use resolvers to define custom
+logic to calculate values of parameters or catalog entries, or inject these values from elsewhere. To use this feature with Kedro, you can pass a
```

Comment:
```suggestion
logic to calculate values of parameters or catalog entries, or inject these values from elsewhere. To use this feature with Kedro, pass a
```

Reply:
Done in 4dd40f7275980317d3d57065fb0f75d36c5b6464